addons/purchase_requisition/models/purchase_requisition.py

```diff
@@ -66,6 +66,25 @@ def _get_type_id(self):
                               copy=False, default='draft')
     account_analytic_id = fields.Many2one('account.analytic.account', 'Analytic Account')
     picking_type_id = fields.Many2one('stock.picking.type', 'Operation Type', required=True, default=_get_picking_in)
+    is_quantity_copy = fields.Boolean(compute='_is_quantity_copy')
```

Comment:
commit message: The witch won't be happy

Reply:
computes should be named <code>_compute_something</code>